### DISCUSSION TITLE
Discord

### DISCUSSION THREAD

**Discussion by irhrhd** [Prodigy-Hacking/ProdigyMathGameHacking#616] [(A) Anything]
@PatheticMustan Watch this then you guys can do this so i can use discord so then i can help and so whatever you guys need https://youtube.com/watch?v=AeU6tR5z3KQ

**Comment by ghost** [comment]
its just basically telling you how to make a discord account

**Reply by irhrhd** [reply]
Oh well me and the proxy developer are working on a fix for discord we already fixed youtube

**Comment by irhrhd** [comment]
Oh cause i thought you could do it so i could talk on the discord server
with that widget bot

**Comment by irhrhd** [comment]
Could you guys try it please i realy want to be able to talk with you guys
